test/tests/ReplayTests.cpp

```diff
@@ -98,8 +98,9 @@ TEST_P(ReplayTests, RunReplay)
     while (replayManager->IsReplaying())
     {
         gs->UpdateLogic();
-        ASSERT_TRUE(replayManager->IsPlaybackStateMismatching() == false);
     }
+    ASSERT_FALSE(replayManager->IsReplaying());
+    ASSERT_FALSE(replayManager->IsPlaybackStateMismatching());
```

Comment:
Is there any point continuing the replay if there has been a mismatch

Reply:
I thought about that as well, I can stop the playback in CheckState I guess.

Reply:
Also is it possible for the checksum to be wrong at the start but then become correct in the subsequent tick? Not sure if that is possible.

Reply:
I don't think that's possible, they are checked after the update.